scripts/loopback-aliases-macos.sh

```diff
@@ -32,7 +32,7 @@ fi
 
 # Function to set aliases
 set_alias() {
-	for i in {2..254}
+	for i in {2..20}
```

Comment:
why this change?

Reply:
If I set till 254, internet stop working. I don't know why, so changed it for now.

Reply:
in that case, add such comment